sar/README.md

```diff
@@ -141,6 +143,7 @@ only additional requirements and mappings to fulfill the CARD4L requirements are
 | card4l:egm_resampling_method           | ✗           | ✓ 4.2       | `EGMResamplingMethod`                                        | string                                                  | The resampling method used for the EGM, see above for example values. |
 | card4l:northern_geometric_accuracy     | ✗           | ✓ 4.3       | `NorthernSTDev` / `NorthernBias` in `GeoCorrAccuracy`        | number                                                  | **REQUIRED.** An estimate of the northern geometric accuracy in meters. |
 | card4l:eastern_geometric_accuracy      | ✗           | ✓ 4.3       | `EasternSTDev` / `EasternBias` in `GeoCorrAccuracy`          | number                                                  | **REQUIRED.** An estimate of the eastern geometric accuracy in meters. |
+| card4l:gridding_convention             | ✗           | ✓ 4.4       | `GriddingConvention`                                         | string                                                  | **REQUIRED.** A brief free text description of the gridding convention used. Can alternatively be provided as a [reference link](#stac-item-links) if available. |
```

Comment:
This is missing in the JSON Schema, please add.

Reply:
I would not add it as a required property in the schema and clarify the description in the following way:

```markdown
**REQUIRED if no reference URL/DOI has been provided as an [Item Link](#stac-item-links)** A brief free text description of the gridding convention used. 
```

Reply:
Works for me, then we need to check in the JSON Schema whether either a link with the rel type is present or the field is given. Maybe list the relation type to use in the description, too.

Reply:
> then we need to check in the JSON Schema whether either a link with the rel type is present or the field is given

Something like this?

```json
 "anyOf": [
    {
      "type": "object",
      "properties": {
        "properties": {
          "type": "object",
          "properties": {
            "card4l:gridding_convention": {
              "type": "string"
            }
          }
        }
      }
    },
    {
      "type": "object",
      "properties": {
        "links": {
          "type": "object",
          "properties": {
            "rel": {
                "const": "gridding-convention"
              }
          }
        }
      }
    }
  ]

```

Reply:
No, you'd need to use contains for the links array and require the card4l:gridding_convention property. And also you need to separate requirements from field validation. But I can fix that later, it's a bit complicated to get that right.

Reply:
Okay, thanks! I've tried to change it a bit, but don't know enough about json schemas to solve it quickly.

df3e091

Reply:
You nearly got it, but "links" is of type array :-)